openmdao/docs/examples/beam_optimization/beam_optimization.rst

```diff
@@ -0,0 +1,83 @@
+.. _`beam_optimization_example`:
+
+Optimizing the thickness distribution of a cantilever beam using the adjoint method
+===================================================================================
+
+In this example, we optimize the thickness (height) distribution of cantilever beam
+using the adjoint method to compute the gradient.
+We use Euler--Bernoulli beam theory and assume a rectangular section.
+
+Background
+----------
+
+The minimize compliance subject to a constraint on the beam volume.
+The optimization problem is
+
+.. math ::
+
+  \begin{array}{r c l}
+    \text{minimize} & & f^T d \\
+    \text{with respect to} & & h \\
+    \text{subject to} & & \text{sum}(h) b L_0 = \text{volume} \\
+  \end{array}
+
+where :math:`f` is the vector of forces, :math:`h` is the vector of beam heights, and :math:`L_0` is the length of a single beam element.
+
+The displacements vector :math:`d` is given by
+
+.. math ::
+
+  K d = f ,
```

Comment:
extraneous comma?

Reply:
This one was intentional - including a comma here is what's normally done in academic papers